test/unit/express-tooling.ts

```diff
@@ -63,16 +65,12 @@ describe('express tooling', () => {
           scope: ['uid', 'test']
         };
         const requiredScopes = ['uid', 'test', 'additional'];
-        let called = false;
-        const next = () => {
-          called = true;
-        };
 
         // when
         requireScopesMiddleware(requiredScopes)(requestMock, responseMock, next);
 
         // then
-        return expect(called).to.be.false;
+        return expect(next).to.not.have.been.called; // async calls won't be detected
```

Comment:
I do not exactly understand the comment here, can you elaborate on that?

Reply:
What I mean is, if there is an asynchronous call to next(), e.g. something like
```javascript
Promise.resolve().then(next)
```
somewhere inside the middleware call, it won't be detected, just as it wouldn't be detected in previous implementation of the test.

Reply:
I see, thanks. So to "fix" this we would have to return the promise from `requireScopesMiddleware`, maybe that is a good idea anyway...

Reply:
It might be a good idea, just beware of unhandled promise rejections.

Reply:
FYI: we created a follow up ticket for this issue #132